template.json

```diff
@@ -1020,6 +1020,7 @@
                 }
             },
             "ability": {
+                "defaultCharacteristicForAbility": "",
```

Comment:
if we add it, let's shorten it a bit: something like defaultChar.
When I first took over the project, everything was in the datamodel, even constants (you can still see remains left and right). That's also when the file metadata.js was created.
To give you an idea, every spell had more that 1KB of useless constant data....
Since then I try to use relatively shot field names.

Ability rolls **have to** have a characteristic and there is one chance out of eight that Intelligence will be the correct one if nothing is configured. That said, **I believe setting it to "int" (ie Intelligence) by default is better** than "" or "n-a" and having to configure it everytime you roll until you edit the ability.

Reply:
Should I keep the N/A or default it to int then?

Reply:
Sorry if I was unclear, default to "int" is better than N/A.

Reply:
Ok, so remove the N/A, put the default to int. Got it